www/index.php

```diff
@@ -237,68 +244,25 @@
 				</div>
 			</div>
 			
-			<div id="setRootNode" class="fieldgroup" style="">
+			<div id="setRootNode" class="fieldgroup" style="display: none;">
```

Comment:
Do you suggest changing whole set of ids in node editor or few in particular? I think it is a good idea , may be @jonwetzel you have any suggestions in this context for naming ids of node editor elements